### PR TITLE
Make ambient warning more extreme

### PR DESCRIPTION
**Pull request by howardjohn**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
LGTM

File: content/en/docs/ops/ambient/getting-started/index.md [hunk 1]
```diff
@@ -6,8 +6,21 @@ weight: 1
...
+
+In the past, smaller "alpha" features have often been introduced without any issues, and later promoted to "stable" without changes.
+This may give users an over-optimistic view of the stability and safety guarantees of an "alpha" feature.
+
+Given the scope of ambient mode, it is imperative to treat the feature as dictated by the feature phase definitions.
+In particular, there are already known performance, stability, and security issues in the alpha release.
+There are also planned breaking changes, including those that will prevent upgrades.
+These are all limitations that will be addressed before graduation to "beta".
+
+Please **do not run ambient in production** and ensure that you understand these caveats during its "alpha" status before use.
```
Comment (frankbu) [inline content/en/docs/ops/ambient/getting-started/index.md:23]: This seems like it's going too far the other way. How about something in between the two extremes?
```suggestion
Ambient is currently in [alpha status](/docs/releases/feature-stages/#feature-phase-definitions).

Please **do not run ambient in production** and be sure to thoroughly review the [feature phase definitions](/docs/releases/feature-stages/#feature-phase-definitions) before use.
In particular, there are already known performance, stability, and security issues in the alpha release.
There are also planned breaking changes, including those that will prevent upgrades.
These are all limitations that will be addressed before graduation to "beta".
```

File: content/en/docs/ops/ambient/getting-started/index.md [hunk 2]
```diff
@@ -6,8 +6,21 @@ weight: 1
...
+Please thoroughly review the [feature phase definitions](/docs/releases/feature-stages/#feature-phase-definitions) before use.
+
+In the past, smaller "alpha" features have often been introduced without any issues, and later promoted to "stable" without changes.
+This may give users an over-optimistic view of the stability and safety guarantees of an "alpha" feature.
+
+Given the scope of ambient mode, it is imperative to treat the feature as dictated by the feature phase definitions.
+In particular, there are already known performance, stability, and security issues in the alpha release.
+There are also planned breaking changes, including those that will prevent upgrades.
+These are all limitations that will be addressed before graduation to "beta".
+
```
Comment (linsun) [inline content/en/docs/ops/ambient/getting-started/index.md:22]: Do we really need to highlight these? per alpha definition which you linked, it is targeted to developers and expert for early feedback, and no guarantees on backward compatibility, may have obvious security vulnerabilities etc.

File: content/en/docs/ops/ambient/getting-started/index.md (hunk 1 above)
Comment (linsun) [reply content/en/docs/ops/ambient/getting-started/index.md:23]: Agreed, it is going too far, i think what you proposed makes sense to me.

**Review by linsun — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

File: content/en/docs/ops/ambient/getting-started/index.md (hunk 2 above)
Comment (howardjohn) [reply content/en/docs/ops/ambient/getting-started/index.md]: Given ~all users run alpha features in prod I doubt it is enough without this.

**Pull request merged**